### DISCUSSION TITLE
How to apply a custom Auth route/guard?

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [filamentphp/filament#7732] [Help]
In version 2.x, you could open config/filament.php and change this line:

```
'auth' => [
    'guard' => env('FILAMENT_AUTH_GUARD', 'filament'),
    'pages' => [
        'login' => \Chiiya\FilamentAccessControl\Http\Livewire\Login::class,
    ],
],
```

How do we do this in Version 3.x?

This is no longer there and putting it there does nothing. How do we change?

**Comment by gboquizosanchez** [comment]
The faster way is on AdminPanelProvider class and add this line to $panel.

`$panel->authGuard('your-guard')`
